src/Tribe/Assets.php

```diff
@@ -207,4 +213,36 @@ public function add_data_strings( $data ) {
 
 		return $data;
 	}
+
+	/**
+	 * Check if we should load the common full style assets.
+	 * When TEC is not in place, or vies V2 are not enabled, so we have the common
+	 * styles we need for our tickets blocks, AR, etc.
+	 * If V2 are active, we respect the style option.
+	 *
+	 * @since  TBD
+	 *
+	 * @return bool
+	 */
+	public function should_enqueue_common_full() {
+		// If TEC isn't there, we need to load common full styles.
+		if ( ! class_exists( 'Tribe__Events__Main' ) ) {
+			return true;
+		}
+
+		// If TEC isn't active or they have a previous version.
+		if ( ! function_exists( 'tribe_events_views_v2_is_enabled' ) ) {
+			return true;
+		}
+
+		// If the views V2 are not enabled, we need to load common full styles.
+		if ( ! tribe_events_views_v2_is_enabled() ) {
+			return true;
+		}
+
+		// If views V2 are in place, we respect the setting.
+		$style_option = tribe_get_option( 'stylesheetOption', 'tribe' );
+		return 'skeleton' !== $style_option;
```

Comment:
Can we use the method from the class in `Tribe\Events\Views\V2\Assets::is_skeleton_style`?
`! tribe( Tribe\Events\Views\V2\Assets::class )->is_skeleton_style()`